conf/distro/nao-core-minimal.conf

```diff
@@ -28,5 +28,5 @@ KERNEL_MODULE_AUTOLOAD += "cgosdrv"
 
 DISTRO_FEATURES:append = " systemd"
 VIRTUAL-RUNTIME_init_manager = "systemd"
-DISTRO_FEATURES_BACKFILL_CONSIDERED += "sysvinit"
-VIRTUAL-RUNTIME_initscript = "systemd-compat-units"
+DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
+VIRTUAL-RUNTIME_initscripts = ""
```

Comment:
Can you please explain why you did these changes?

Reply:
Copied from here: https://docs.yoctoproject.org/dev/dev-manual/init-manager.html#using-systemd-exclusively `initscripts` seemed to be a typo previously.